lab-glen/scss/main.scss

```diff
@@ -0,0 +1,76 @@
+// :::: $vars ::::
+
+$brand-primary: #444;
+$btn-primary: $brand-primary / 2;
+$btn-hover: #31b1c1;
+$black: #000;
+$white: #fff;
+$box-std: 40%;
+$gutter-std: 5%;
+$gutter-sm: $gutter-std / 2;
+
+// :::: global styles ::::
+
+body {
+  font-family: arial;
+  font-size: 1vw;
+  background: $brand-primary * 3;
+}
+
+h2 {
+  font-size: 2vw;
+  text-align: center;
+  margin-bottom: $gutter-std;
+}
```

Comment:
I want to see you start playing around with more of your styles. Add some flavor. Days like this for lab you can start crushing some other things.